R/reportEmi.R

```diff
@@ -1769,6 +1810,24 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL,
                # total DACCS
                setNames(-out[, , "Carbon Management|Storage|+|DAC (Mt CO2/yr)"],
                         "Emi|CO2|CDR|DACCS (Mt CO2/yr)"),
+
+               # gross OAE
+               setNames(vm_emiCdrTeDetail[, , "oae_ng"] * GtC_2_MtCO2,
+                        "Emi|CO2|CDR|OAE|+|Ocean Uptake, traditional calciner (Mt CO2/yr)"),
+               setNames(vm_emiCdrTeDetail[, , "oae_el"] * GtC_2_MtCO2,
+                        "Emi|CO2|CDR|OAE|+|Ocean Uptake, electric calciner (Mt CO2/yr)"),
+               setNames((vm_emiCdrTeDetail[, , "oae_ng"] + vm_emiCdrTeDetail[, , "oae_el"]) * GtC_2_MtCO2,
+                        "Emi|CO2|CDR|OAE|+|Ocean Uptake (Mt CO2/yr)"),
+               # OAE process emissions
+               setNames(out[, , "Carbon Management|Carbon Capture|+|OAE calcination (Mt CO2/yr)"]
+                        - out[, , "Carbon Management|Storage|+|OAE calcination (Mt CO2/yr)"],
+                        "Emi|CO2|CDR|OAE|+|Calcination emissions (Mt CO2/yr)"),
```

Comment:
add emissions that are not captured

Reply:
I created a new variable for this: 'Emi|CO2|CDR|net OAE|+|Unavoidable emissions (Mt CO2/yr)'